app/views/hackathons/submissions/new.html.erb

```diff
@@ -1,25 +1,25 @@
-<% content_for :title, "Apply to list your hackathon" %>
+<% content_for :title, "List your hackathon on hackathons.hackclub.com" %>
```

Comment:
feels weird to be applying, as long as you meet the requirements your in... idk i might be the one who cares about this sorta thing

Reply:
I think it reflects the fact that it needs approval - you're not immediately on the site.

Reply:
I think we make that clear in the description, making it an application feels like this is some sort of exclusive list... at least that's my reading of the word

Reply:
If it's below, then I'm fine with this.

Reply:
I'm cool with this!